### DISCUSSION TITLE
Is there a way to make words not repeat?

### DISCUSSION THREAD

**Discussion by tvvoty** [monkeytypegame/monkeytype#1497] [Questions]
Basically, if you do a 10000 words list in words mode, can you make so that if you type a word once it won't appear in the future, thus definitely going through every word in the list at least once?

**Comment by unrealapex** [comment]
That depends on the language. For example if you used the `english` language for the test , there are going to be repeating words because there are only 200 words in the file. If you are doing the `english_10k` there probably would still be some repeated words but less and so on.

**Comment by frogfrogfrogfrogfrog** [comment]
I think the only way you could recreate this is either find a random list of the top 10,000 words or go into the files and find what 10,000 words are in the english_10k mode then copy and paste that into custom mode and don't turn on random.

**Reply by tvvoty** [reply]
That works! Thanks a lot for the answer! <3
